src/targets/crazybee_f4_frsky/target.h

```diff
@@ -14,11 +14,36 @@
   USART2_PA3PA2
 
 // LEDS
-#define LED_NUMBER 1
+#define LED_NUMBER 2
 #define LED1PIN PIN_C13
 #define LED1_INVERT
+#define LED2PIN PIN_B9
+#define LED2_INVERT
 #define BUZZER_PIN PIN_C15
 
+//--------------------------------------------------------------
+// RGB LEDs
+//--------------------------------------------------------------
+#define RGB_LED_NUMBER 2
+//--------------------------------------------------------------
+// DMA => TIM2-CC1 => DMA1, Channel3, Stream5 (See drv_dma.c)
+//--------------------------------------------------------------
+#define RGB_LED_DMA       DMA_DEVICE_TIM2_CH1
+//--------------------------------------------------------------
+// pin / port for the RGB led
+//--------------------------------------------------------------
+#define RGB_PIN LL_GPIO_PIN_0
+#define RGB_PORT GPIOA
```

Comment:
please use the `PIN_A0` method of specifying these, that will also allow the drv_gpio helpers to work.